servers/lib/test/README.md

```diff
@@ -0,0 +1,31 @@
+# Testing File Structure
+
+Inorder to test the lib-MS, a specific file structure must be configured.
+
+Since the tests are tested across two different modes, 'local' or 'gitlab', a homogenous file structure is required.
+
+This file structure must be within 'files' aswell as within your working gitlab repository, as depicted below...
+
+local file system:
+
+```
+files/
+  user2/
+    Data/
```

Comment:
why can't all these be lower case. If you absolutely need these to be upper case, then you need to update script/test.bash to do the task of changing the directory names in files/user2 and then reverting the structure at the end of the integration test.

Reply:
There is no necessary reason, so i have changed it all to lowercase - sorry about that